packages/cli/src/__tests__/fix.ts

```diff
@@ -70,6 +70,78 @@ test("set main and module field", async () => {
   `);
 });
 
+test("set exports field when opt-in", async () => {
+  let tmpPath = f.copy("package-exports");
+
+  await fix(tmpPath);
+
+  let pkg = await getPkg(tmpPath);
+
+  // Assert that the order of conditions are correct.
+  const conditionsOrder = Object.keys(pkg.exports["."]);
+  expect(conditionsOrder).toEqual([
+    "worker",
+    "browser",
+    "production",
+    "module",
+    "default",
+  ]);
+  const subConditionsOrder = Object.keys(pkg.exports["."].browser);
+  expect(subConditionsOrder).toEqual(["production", "module", "default"]);
+
+  // NOTE: The order of the conditions is important and JEST is sorting the keys alphabetically.
+  // The tests above actually assert that the order is correct.
+  expect(pkg).toMatchInlineSnapshot(`
```

Comment:
Couldn't all of those assertions here be replaced with this (and a comment about the order):
```js
expect(JSON.stringify(pkg, null, 2)).toMatchInlineSnapshot()
```

Jest can't sort the keys within a string, right? 😉

Reply:
Clever, I can make that change.